test/test_out_s3.rb

```diff
@@ -427,7 +427,7 @@ def setup_mocks(exists_return = false)
     mock(Aws::S3::Client).new(anything).at_least(0) { @s3_client }
     @s3_resource = mock(Aws::S3::Resource.new(client: @s3_client))
     mock(Aws::S3::Resource).new(client: @s3_client) { @s3_resource }
-    @s3_bucket = mock(Aws::S3::Bucket.new(name: "test",
+    @s3_bucket = mock(Aws::S3::Bucket.new(name: "test_bucket",
```

Comment:
Why this change is needed?
To get merged this PR, we should add placeholders testcases.

Reply:
I sincerely don't know. I thought someone changed it last patch. It was producing an error on tests I guess. I will check it later.

Yeah... I was afraid you would say that about testing placeholders. I truly have no idea how to do it right now since they're evaluated during plug-in execution time.

Any suggestions? If someone give a hint I might get through it. I am no expert on ruby tests though.

Reply:
> Yeah... I was afraid you would say that about testing placeholders. I truly have no idea how to do it right now since they're evaluated during plug-in execution time.

These testcases are fluent-plugin-elasticsearch's but they shows how to write placeholders testcases:
https://github.com/uken/fluent-plugin-elasticsearch/blob/c0a4713edfc620cf566ede02f0f85930d1c5a88a/test/plugin/test_out_elasticsearch.rb#L3893

* Write configuration which includes `<buffer attributes1, attributes2, ..., attributesN>` with `Fluent::Config::Element.new` and using tag/time/user defined(variables) placeholders
* Creates chunk metadata keys which are originated from `<buffer attributes1, attributes2, ..., attributesN>` (This is automatically created when use test driver#feed)
* Assert placeholders replaced values with test-unit assertions